ddev/src/ddev/repo/constants.py

```diff
@@ -3,6 +3,12 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 CONFIG_DIRECTORY = '.ddev'
 NOT_SHIPPABLE = frozenset(['datadog_checks_dev', 'datadog_checks_tests_helper', 'ddev'])
+FULL_NAMES = {
+    'core': 'integrations-core',
+    'extras': 'integrations-extras',
+    'marketplace': 'marketplace',
+    'agent': 'datadog-agent',
+}
```

Comment:
should we include `internal`?

Reply:
I thought we shouldn't since it is just specific to our people and also there is no equivalent top level `ddev` command line switch for that

Reply:
sounds good!